cli/azd/pkg/environment/environment.go

```diff
@@ -46,6 +46,9 @@ const ResourceGroupEnvVarName = "AZURE_RESOURCE_GROUP"
 // PlatformTypeEnvVarName is the name of the key used to store the current azd platform type
 const PlatformTypeEnvVarName = "AZD_PLATFORM_TYPE"
 
+// DisablePreflightName is used to allow users to skip the preflight validation check
+const DisablePreflightName = "AZURE_PROVISION_SKIP_VALIDATE"
```

Comment:
Consider adding this env var to: https://github.com/Azure/azure-dev/blob/main/cli/azd/docs/environment-variables.md